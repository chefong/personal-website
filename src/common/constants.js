import hamburgerMenuAnimation from '~/assets/menu-animation.json';

export const gaEvents = {
    PAGE_RENDERED: 'page_rendered',
    ABOUT_NAV_CLICKED: 'about_nav_clicked',
    PROJECTS_NAV_CLICKED: 'projects_nav_clicked',
    EXPERIENCE_NAV_CLICKED: 'experience_nav_clicked',
    THEME_TOGGLE_CLICKED: 'theme_toggle_clicked',
    RESUME_CLICKED: 'resume_clicked',
    CONTACT_ME_CLICKED: 'contact_me_clicked',
    GITHUB_CLICKED: 'github_icon_clicked',
    LINKEDIN_CLICKED: 'linkedin_icon_clicked',
    SPOTIFY_CLICKED: 'spotify_icon_clicked',
    MEDIUM_CLICKED: 'medium_icon_clicked',
    MAIL_CLICKED: 'mail_icon_clicked',
};

export const navItems = [
    {
        to: 'about',
        num: '01.',
        name: 'About',
        clickEventId: gaEvents.ABOUT_NAV_CLICKED,
    },
    {
        to: 'projects',
        num: '02.',
        name: 'Projects',
        clickEventId: gaEvents.PROJECTS_NAV_CLICKED,
    },
    {
        to: 'experience',
        num: '03.',
        name: 'Experience',
        clickEventId: gaEvents.EXPERIENCE_NAV_CLICKED,
    },
];

export const projects = [
    {
        nameId: 'PROJECTS_PATCH',
        descriptionId: 'PROJECTS_PATCH_DESCRIPTION',
        githubLink: 'https://github.com/willshiao/Patch',
    },
    {
        nameId: 'PROJECTS_CHEATERBEATER',
        descriptionId: 'PROJECTS_CHEATERBEATER_DESCRIPTION',
        githubLink: 'https://github.com/willshiao/CheaterBeater',
    },
    {
        nameId: 'PROJECTS_BAUT',
        descriptionId: 'PROJECTS_BAUT_DESCRIPTION',
        githubLink: 'https://github.com/KirbyDownB/GEICOChatBot',
    },
    {
        nameId: 'PROJECTS_NUTSHELL',
        descriptionId: 'PROJECTS_NUTSHELL_DESCRIPTION',
        githubLink: 'https://github.com/chefong/nutshell',
    },
    {
        nameId: 'PROJECTS_STOCK_OVERFLOW',
        descriptionId: 'PROJECTS_STOCK_OVERFLOW_DESCRIPTION',
        githubLink: 'https://github.com/kimjihwan0208/stockoverflow',
    },
    {
        nameId: 'PROJECTS_JOYRIDE',
        descriptionId: 'PROJECTS_JOYRIDE_DESCRIPTION',
        githubLink: 'https://github.com/ericong18/joyride',
    },
];

export const experiences = [
    {
        imagePath: {
            light: 'imgs/experience/opentable.png',
            dark: 'imgs/experience/opentable-white.png',
        },
        nameId: 'EXPERIENCE_OPENTABLE',
        positionId: 'SOFTWARE_ENGINEER_1',
        duration: 'February 2021 - Present',
        teamId: 'EXPERIENCE_OPENTABLE_TEAM_2',
        initiativesId: 'EXPERIENCE_OPENTABLE_INITIATIVES_2',
        technologiesId: 'EXPERIENCE_OPENTABLE_TECHNOLOGIES_2',
    },
    {
        imagePath: {
            light: '/imgs/experience/intuit.svg',
            dark: '/imgs/experience/intuit-white.svg',
        },
        nameId: 'EXPERIENCE_INTUIT',
        positionId: 'SOFTWARE_ENGINEER_1',
        duration: 'July 2020 - February 2021',
        teamId: 'EXPERIENCE_INTUIT_TEAM',
        initiativesId: 'EXPERIENCE_INTUIT_INITIATIVES',
        technologiesId: 'EXPERIENCE_INTUIT_TECHNOLOGIES',
    },
    {
        imagePath: {
            light: 'imgs/experience/opentable.png',
            dark: 'imgs/experience/opentable-white.png',
        },
        nameId: 'EXPERIENCE_OPENTABLE',
        positionId: 'SOFTWARE_ENGINEERING_INTERN',
        duration: 'June 2019 - August 2019',
        teamId: 'EXPERIENCE_OPENTABLE_TEAM_1',
        initiativesId: 'EXPERIENCE_OPENTABLE_INITIATIVES_1',
        technologiesId: 'EXPERIENCE_OPENTABLE_TECHNOLOGIES_1',
    },
];

export const handles = [
    {
        name: 'GitHub',
        icon: ['fab', 'github'],
        link: 'https://github.com/chefong',
        clickEventId: gaEvents.GITHUB_CLICKED,
    },
    {
        name: 'LinkedIn',
        icon: ['fab', 'linkedin-in'],
        link: 'https://www.linkedin.com/in/ericong18/',
        clickEventId: gaEvents.LINKEDIN_CLICKED,
    },
    {
        name: 'Spotify',
        icon: ['fab', 'spotify'],
        link: 'https://open.spotify.com/user/ericong18?si=H0kHkFdxQ-GyKIlfLWzDqg',
        clickEventId: gaEvents.SPOTIFY_CLICKED,
    },
    {
        name: 'Medium',
        icon: ['fab', 'medium'],
        link: 'https://medium.com/@ericong18',
        clickEventId: gaEvents.MEDIUM_CLICKED,
    },
    {
        name: 'Email',
        icon: 'envelope',
        link: 'mailto:ericong18@gmail.com',
        clickEventId: gaEvents.MAIL_CLICKED,
    },
];

export const likes = [
    'ABOUT_LIKE_TEA',
    'ABOUT_LIKE_WATERMELON',
    'ABOUT_LIKE_MUSIC',
    'ABOUT_LIKE_DESIGN',
    'ABOUT_LIKE_PREMIER_LEAGUE',
];

export const dislikes = ['ABOUT_DISLIKE_CILANTRO'];

export const themeColors = {
    light: '#fffefb',
    dark: '#222222',
    tan: '#f1e7d0',
};

export const lottieProps = {
    loop: false,
    autoplay: false,
    animationData: hamburgerMenuAnimation,
};

export const reactScrollLinkProps = {
    smooth: true,
    offset: -150,
    duration: 750,
};

// Segment frames for menu open and close
export const menuOpenSegments = [30, 70];
export const menuCloseSegments = [100, 140];

// Box shadow styling for when the nav menu is open
export const menuBoxShadow = '0px 4px 28px rgba(0, 0, 0, 0.06)';

// Menu transition styling for menu
export const menuTransition = 'all 0.5s ease';

export const themeOrder = ['light', 'tan', 'dark'];

export const iconSize = '28px';

export const iconColors = {
    light: themeColors.dark,
    dark: themeColors.light,
    tan: themeColors.dark,
};

export const toastOptions = {
    style: {
        padding: '16px',
        fontFamily: 'Greycliff',
        backgroundColor: '#303030',
        color: 'white',
    },
};

export const particlesParams = {
    particles: {
        number: {
            value: 80,
            density: {
                enable: true,
                value_area: 1500,
            },
        },
        line_linked: {
            enable: true,
            opacity: 0.02,
        },
        move: {
            direction: 'right',
            speed: 0.05,
        },
        size: {
            value: 1.25,
        },
        opacity: {
            anim: {
                enable: true,
                speed: 1,
                opacity_min: 0.05,
            },
        },
    },
    retina_detect: true,
};
