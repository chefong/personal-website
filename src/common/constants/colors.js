export default {
    light: '#fffefb',
    dark: '#222222',
    tan: '#f1e7d0',
};
