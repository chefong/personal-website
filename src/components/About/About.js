import styles from './About.module.css';

export default function About() {
  return (
    <section className={`${styles.section} container-fluid`} id="about">
      <div className="row justify-content-center">
        <div className="col-lg-5 col-md-6">
          <div className={styles.headerGroup}>
            <h2 className={styles.header}><span className={styles.headerNum}>01.</span> About Me</h2>
          </div>
          <div className={styles.content}>
            <p className={styles.description}>I graduated Magna Cum Laude from the <a href="https://www.ucr.edu/" className={styles.link} target="_blank">University of California, Riverside</a> in June 2020 with a B.S. degree in Computer Science. I’m an incoming Software Engineer at <a href="https://www.intuit.com/" className={styles.link} target="_blank">Intuit</a>, where I'll be working within the Financial Data Platform Team. Previously, I was at <a href="https://www.opentable.com/" className={styles.link} target="_blank">OpenTable</a> as a Software Engineering Intern on the Content Team.</p>
            <div className={styles.thingsGroup}>
              <div className="row justify-content-center">
                <div className="col-6">
                  <p className={styles.thingsHeader}>Things I Like</p>
                  <ul className={styles.thingsList}>
                    <li>Watermelon 🍉</li>
                    <li>70 - 85 BPM music 🎶</li>
                    <li>Minimal design 🎨</li>
                    <li>Premier League ⚽</li>
                  </ul>
                </div>
                <div className="col-6">
                  <p className={styles.thingsHeader}>Thing I Don't Like</p>
                  <ul className={styles.thingsList}>
                    <li>Cilantro 🤢</li>
                  </ul>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div className="col-lg-3 col-md-4">
          <div className={styles.imageContainer}>
            <img src="/imgs/about/spin.gif" className={styles.image} alt=""/>
          </div>
        </div>
      </div>
    </section>
  );
}