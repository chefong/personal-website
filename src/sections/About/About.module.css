.section {
  margin-top: 6em;
}

.link {
  font-weight: 500;
  color: #4A74E1;
}

.link:hover {
  color: #4A74E1;
}

.headerGroup {
  font-weight: 500;
}

.headerNum {
  color: #4A74E1;
}

.thingsGroup {
  margin-top: 1.5em;
}

.thingsHeader {
  font-weight: 500;
  color: #303030;
}

.thingsList {
  margin-top: -8px;
  margin-left: -20px;
}

@keyframes spin {
  from {
    transform: rotate(0deg);
  }
  to {
    transform: rotate(360deg);
  }
}

.image {
  width: 100%;
  border-radius: 50%;
  margin-top: 2em;
  padding: 1em;
  pointer-events: none;
}