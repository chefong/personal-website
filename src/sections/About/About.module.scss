.section {
  margin: 10em auto;
}

.link {
  font-weight: 500;
  color: var(--theme-color);
}

.link:hover {
  color: var(--theme-color);
}

.header {
  color: var(--theme-page-text);
}

.headerGroup {
  font-weight: 500;
}

.headerNum {
  color: var(--theme-color);
}

.thingsGroup {
  margin-top: 1.5em;
}

.thingsHeader {
  font-weight: 500;
  color: var(--theme-page-text);
}

.thingsList {
  margin-top: -8px;
  margin-left: -20px;
  color: var(--theme-page-text);
}

.description {
  color: var(--theme-page-text);
}

.hero {
  width: 75%;
  display: block;
  margin: 0 auto;
  margin-top: 1em;
  transition: all 0.25s ease;
  filter: saturate(0);
  border-radius: 50%;
  border: 8px solid var(--theme-color);
}

.hero:hover {
  filter: saturate(1);
}
