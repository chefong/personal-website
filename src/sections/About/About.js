import styles from './About.module.css';
import { Fade } from 'react-reveal';
import { revealProps, likedThings, dislikedThings } from '../../shared/constants';
import { FormattedMessage } from 'text-provider';
import Lottie from 'lottie-react';
import codeAnimation from './code-animation.json';

const lottieProps = {
  loop: true,
  autoplay: true,
  animationData: codeAnimation
};

export default function About() {
  return (
    <section className={`${styles.section} container-fluid`} id="about">
      <div className="row justify-content-center">
        <div className="col-lg-5 col-md-6">
          <div className={styles.headerGroup}>
            <Fade {...revealProps}>
              <h2 className={styles.header}>
                <span className={styles.headerNum}>01. </span> 
                <FormattedMessage id="ABOUT_HEADING" />
                </h2>
            </Fade>
          </div>
          <div className={styles.content}>
            <Fade {...revealProps} cascade>
              <p className={styles.description}>
                <FormattedMessage
                  id="ABOUT_DESCRIPTION"
                  values={{
                    schoolClass: styles.link,
                    intuitClass: styles.link,
                    openTableClass: styles.link
                  }}
                />
              </p>
            </Fade>
            <div className={styles.thingsGroup}>
              <div className="row justify-content-center">
                <div className="col-6">
                  <Fade {...revealProps}>
                    <p className={styles.thingsHeader}>
                      <FormattedMessage id="ABOUT_LIKE" />
                    </p>
                  </Fade>
                  <Fade {...revealProps} cascade>
                    <ul className={styles.thingsList}>
                      {likedThings.map(id => (
                        <li key={id}>
                          <FormattedMessage id={id} />
                        </li>
                      ))}
                    </ul>
                  </Fade>
                </div>
                <div className="col-6">
                  <Fade {...revealProps}>
                    <p className={styles.thingsHeader}>
                      <FormattedMessage id="ABOUT_DISLIKE" />
                    </p>
                  </Fade>
                  <Fade {...revealProps} cascade>
                    <ul className={styles.thingsList}>
                      {dislikedThings.map(id => (
                        <li key={id}>
                          <FormattedMessage id={id} />
                        </li>
                      ))}
                    </ul>
                  </Fade>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div className="col-lg-3 col-md-4">
          <div className={styles.imageContainer}>
            <Lottie {...lottieProps} />
          </div>
        </div>
      </div>
    </section>
  );
}