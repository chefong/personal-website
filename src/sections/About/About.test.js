import React from 'react';
import { render } from '@testing-library/react';
import About from './About';
import { likes, dislikes } from '../../shared/constants';

describe('About', () => {
  it('renders', () => {
    const { getByTestId } = render(<About />);
    expect(getByTestId('About-container')).toBeDefined();
  });

  it('matches snapshot', () => {
    const { container } = render(<About />);
    expect(container).toMatchSnapshot();
  });

  it('contains correct number of likes', () => {
    const { getAllByTestId } = render(<About />);
    expect(getAllByTestId('About-like')).toHaveLength(likes.length);
  });

  it('contains correct number of dislikes', () => {
    const { getAllByTestId } = render(<About />);
    expect(getAllByTestId('About-dislike')).toHaveLength(dislikes.length);
  });
});