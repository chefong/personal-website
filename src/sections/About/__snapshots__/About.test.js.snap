// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About matches snapshot 1`] = `
<div>
  <section
    class="undefined container-fluid"
    data-testid="About-container"
    id="about"
  >
    <div
      class="row justify-content-center"
    >
      <div
        class="col-lg-5 col-md-6"
      >
        <div>
          <h2
            class="react-reveal"
          >
            <span>
              01. 
            </span>
            <span />
          </h2>
        </div>
        <div>
          <p
            class="react-reveal"
          >
            <span />
          </p>
          <div>
            <div
              class="row justify-content-center"
            >
              <div
                class="col-6"
              >
                <p
                  class="react-reveal"
                >
                  <span />
                </p>
                <ul
                  class="react-reveal"
                >
                  <li
                    data-testid="About-like"
                  >
                    <span />
                  </li>
                  <li
                    data-testid="About-like"
                  >
                    <span />
                  </li>
                  <li
                    data-testid="About-like"
                  >
                    <span />
                  </li>
                  <li
                    data-testid="About-like"
                  >
                    <span />
                  </li>
                </ul>
              </div>
              <div
                class="col-6"
              >
                <p
                  class="react-reveal"
                >
                  <span />
                </p>
                <ul
                  class="react-reveal"
                >
                  <li
                    data-testid="About-dislike"
                  >
                    <span />
                  </li>
                </ul>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        class="col-lg-3 col-md-4"
      >
        <div>
          <div>
            <svg
              height="640"
              preserveAspectRatio="xMidYMid meet"
              style="width: 100%; height: 100%; transform: translate3d(0,0,0);"
              viewBox="0 0 640 640"
              width="640"
              xmlns="http://www.w3.org/2000/svg"
            >
              <defs>
                <clippath
                  id="__lottie_element_74"
                >
                  <rect
                    height="640"
                    width="640"
                    x="0"
                    y="0"
                  />
                </clippath>
                <clippath
                  id="__lottie_element_76"
                >
                  <path
                    d="M0,0 L398,0 L398,380 L0,380z"
                  />
                </clippath>
                <clippath
                  id="__lottie_element_80"
                >
                  <path
                    d="M0,0 L398,0 L398,380 L0,380z"
                  />
                </clippath>
                <clippath
                  id="__lottie_element_84"
                >
                  <path
                    d="M0,0 L398,0 L398,380 L0,380z"
                  />
                </clippath>
                <clippath
                  id="__lottie_element_88"
                >
                  <path
                    clip-rule="nonzero"
                    fill="#ffffff"
                  />
                </clippath>
                <clippath
                  id="__lottie_element_91"
                >
                  <path
                    clip-rule="nonzero"
                    fill="#ffffff"
                  />
                </clippath>
                <clippath
                  id="__lottie_element_94"
                >
                  <path
                    clip-rule="nonzero"
                    fill="#ffffff"
                  />
                </clippath>
                <clippath
                  id="__lottie_element_97"
                >
                  <path
                    clip-rule="nonzero"
                    fill="#ffffff"
                  />
                </clippath>
                <clippath
                  id="__lottie_element_100"
                >
                  <path
                    clip-rule="nonzero"
                    fill="#ffffff"
                  />
                </clippath>
                <clippath
                  id="__lottie_element_103"
                >
                  <path
                    clip-rule="nonzero"
                    fill="#ffffff"
                  />
                </clippath>
                <clippath
                  id="__lottie_element_106"
                >
                  <path
                    clip-rule="nonzero"
                    fill="#ffffff"
                  />
                </clippath>
                <clippath
                  id="__lottie_element_109"
                >
                  <path
                    clip-rule="nonzero"
                    fill="#ffffff"
                  />
                </clippath>
                <clippath
                  id="__lottie_element_112"
                >
                  <path
                    clip-rule="nonzero"
                    fill="#ffffff"
                  />
                </clippath>
                <clippath
                  id="__lottie_element_115"
                >
                  <path
                    clip-rule="nonzero"
                    fill="#ffffff"
                  />
                </clippath>
                <clippath
                  id="__lottie_element_118"
                >
                  <path
                    clip-rule="nonzero"
                    fill="#ffffff"
                  />
                </clippath>
                <clippath
                  id="__lottie_element_121"
                >
                  <path
                    clip-rule="nonzero"
                    fill="#ffffff"
                  />
                </clippath>
                <clippath
                  id="__lottie_element_124"
                >
                  <path
                    clip-rule="nonzero"
                    fill="#ffffff"
                  />
                </clippath>
                <mask
                  id="__lottie_element_127"
                >
                  <path
                    clip-rule="nonzero"
                    fill="#ffffff"
                  />
                </mask>
                <mask
                  id="__lottie_element_130"
                >
                  <path
                    clip-rule="nonzero"
                    fill="#ffffff"
                  />
                </mask>
                <clippath
                  id="__lottie_element_136"
                >
                  <path
                    d="M0,0 L398,0 L398,380 L0,380z"
                  />
                </clippath>
              </defs>
              <g
                clip-path="url(#__lottie_element_74)"
              >
                <g
                  clip-path="url(#__lottie_element_76)"
                  opacity="1"
                  style="display: block;"
                  transform="matrix(1.2799999713897705,0,0,1.2799999713897705,65.27999877929688,76.80000305175781)"
                >
                  <g
                    clip-path="url(#__lottie_element_136)"
                    opacity="1"
                    style="display: block;"
                    transform="matrix(1,0,0,1,0,0)"
                  >
                    <g
                      opacity="1"
                      style="display: block;"
                      transform="matrix(0.7900000214576721,0,0,0.7900000214576721,34.65367126464844,50.18421936035156)"
                    >
                      <g
                        opacity="1"
                        transform="matrix(1,0,0,1,196.5,191.88499450683594)"
                      >
                        <path
                          d=" M-189.23300170898438,2.6419999599456787 C-188.3000030517578,-2.490000009536743 -187.47500610351562,-7.650000095367432 -186.64500427246094,-12.817999839782715 C-184.15499877929688,-28.320999145507812 -181.6179962158203,-43.88100051879883 -175.9709930419922,-58.86899948120117 C-174.18800354003906,-64.4260025024414 -172.19900512695312,-69.81800079345703 -169.97300720214844,-75.08599853515625 C-163.29600524902344,-90.87300109863281 -154.48599243164062,-105.53800201416016 -143.906005859375,-118.72100067138672 C-139.6739959716797,-123.99400329589844 -135.15199279785156,-129.00100708007812 -130.3560028076172,-133.60400390625 C-90.41999816894531,-161.3509979248047 -41.375999450683594,-189.40699768066406 7.11899995803833,-191.35800170898438 C14.04699993133545,-191.63699340820312 20.964000701904297,-191.3820037841797 27.840999603271484,-190.51800537109375 C58.5,-179.88400268554688 91.5,-179.88499450683594 124.5,-176.88499450683594 C152.5,-173.88499450683594 170.5,-150.88499450683594 172.5,-124.88500213623047 C175.5,-96.88500213623047 186.5,-71.88500213623047 196.5,-45.8849983215332 C201.5,-33.8849983215332 201.5,-20.885000228881836 201.33700561523438,-8.069000244140625 C201.13299560546875,-1.1749999523162842 200.5399932861328,5.735000133514404 199.56700134277344,12.628000259399414 C197.6219940185547,26.413999557495117 194.15199279785156,40.13800048828125 189.31500244140625,53.39400100708008 C187.76499938964844,59.52799987792969 185.9759979248047,65.49299621582031 183.81199645996094,71.32499694824219 C172.98399353027344,100.46600341796875 152.73300170898438,126.31300354003906 122.18499755859375,145.0260009765625 C116.25700378417969,149.86399841308594 110.59200286865234,154.49600219726562 104.7300033569336,158.7570037841797 C75.45500183105469,180.10000610351562 39.827999114990234,191.63699340820312 0.7760000228881836,187.00900268554688 C-5.105999946594238,187.49099731445312 -10.866999626159668,187.61500549316406 -16.547000885009766,187.36500549316406 C-22.226999282836914,187.11500549316406 -27.825000762939453,186.49099731445312 -33.35200119018555,185.5330047607422 C-76.5,173.11500549316406 -120.5,173.11500549316406 -162.5,156.11500549316406 C-201.5,140.11500549316406 -183.5,91.11599731445312 -188.197998046875,56.95100021362305 C-189.43299865722656,50.43000030517578 -190.67799377441406,43.959999084472656 -191.31900024414062,37.334999084472656 C-191.96099853515625,30.709999084472656 -191.86599731445312,23.884000778198242 -190.91799926757812,16.85099983215332 C-190.5760040283203,12.17300033569336 -190.00100708007812,7.423999786376953 -189.23300170898438,2.6419999599456787z"
                          fill="rgb(199,214,253)"
                          fill-opacity="1"
                        />
                      </g>
                    </g>
                    <g
                      opacity="1"
                      style="display: block;"
                      transform="matrix(0.7900000214576721,0,0,0.7900000214576721,32.8304443359375,50.77348327636719)"
                    >
                      <g
                        opacity="1"
                        transform="matrix(1,0,0,1,187.30499267578125,192.95899963378906)"
                      >
                        <path
                          d=" M-136.06399536132812,-79.49800109863281 C-133.23399353027344,-83.03399658203125 -130.47500610351562,-86.63800048828125 -127.70899963378906,-90.24400329589844 C-119.41200256347656,-101.06300354003906 -93.16200256347656,-127.84100341796875 -89.28900146484375,-130.947998046875 C-77.6760025024414,-140.2570037841797 -64.89600372314453,-147.8300018310547 -51.374000549316406,-153.5399932861328 C-45.96500015258789,-155.82400512695312 -40.44499969482422,-157.78399658203125 -34.88199996948242,-159.322998046875 C7.078000068664551,-164.0540008544922 56.12300109863281,-165.26600646972656 93.98400115966797,-146.75599670410156 C99.39299774169922,-144.11199951171875 104.572998046875,-141.06500244140625 109.47200012207031,-137.5679931640625 C128.51400756835938,-116.7979965209961 153.72900390625,-103.18699645996094 177.70799255371094,-87.28500366210938 C197.86599731445312,-73.44499969482422 202.13499450683594,-48.44599914550781 192.94000244140625,-27.753999710083008 C183.6840057373047,-5.120999813079834 181.7790069580078,18.51799964904785 178.69700622558594,42.50899887084961 C177.5679931640625,53.7400016784668 172.20599365234375,63.67399978637695 166.79600524902344,73.4000015258789 C163.7969970703125,78.58399963378906 160.49400329589844,83.61900329589844 156.9080047607422,88.48500061035156 C149.73599243164062,98.21700286865234 141.4239959716797,107.27300262451172 132.26100158691406,115.40699768066406 C128.54600524902344,119.45500183105469 124.71900177001953,123.2750015258789 120.66000366210938,126.83899688720703 C100.36699676513672,144.64100646972656 74.23400115966797,156.03799438476562 43.17399978637695,157.73800659179688 C36.64899826049805,158.99000549316406 30.40999984741211,160.19200134277344 24.17300033569336,161.031005859375 C-6.999000072479248,165.26600646972656 -38.97999954223633,159.38699340820312 -66.91200256347656,139.7449951171875 C-71.6050033569336,137.68800354003906 -76.05899810791016,135.40699768066406 -80.2959976196289,132.8730010986328 C-84.53299713134766,130.33900451660156 -88.5530014038086,127.55400085449219 -92.38099670410156,124.54299926757812 C-120.2300033569336,97.25800323486328 -153.8509979248047,79.11100006103516 -178.93299865722656,48.79899978637695 C-202.13499450683594,20.488000869750977 -168.17100524902344,-9.529000282287598 -157.67100524902344,-37.573001861572266 C-155.9250030517578,-43.064998626708984 -154.20700073242188,-48.52199935913086 -151.96499633789062,-53.8489990234375 C-149.72300720214844,-59.17599868774414 -146.83599853515625,-64.35199737548828 -143.21099853515625,-69.33499908447266 C-141.02000427246094,-72.76799774169922 -138.6230010986328,-76.16100311279297 -136.06399536132812,-79.49800109863281z"
                          fill="rgb(74,116,225)"
                          fill-opacity="1"
                        />
                      </g>
                    </g>
                  </g>
                  <g
                    clip-path="url(#__lottie_element_80)"
                    opacity="1"
                    style="display: block;"
                    transform="matrix(0.7200000286102295,0,0,0.7200000286102295,55.720001220703125,53.19999694824219)"
                  >
                    <g
                      opacity="1"
                      style="display: block;"
                      transform="matrix(1,0,0,1,0,0)"
                    >
                      <g
                        opacity="1"
                        transform="matrix(1,0,0,1,192.0399932861328,119.59300231933594)"
                      >
                        <path
                          d=" M115.95999908447266,-1.0130000114440918 C115.95999908447266,-1.0130000114440918 115.95999908447266,9.449999809265137 115.95999908447266,9.449999809265137 C115.95999908447266,9.449999809265137 -115.95999908447266,9.449999809265137 -115.95999908447266,9.449999809265137 C-115.95999908447266,9.449999809265137 -115.95999908447266,-1.0130000114440918 -115.95999908447266,-1.0130000114440918 C-115.95999908447266,-5.671999931335449 -112.18299865722656,-9.449999809265137 -107.52400207519531,-9.449999809265137 C-107.52400207519531,-9.449999809265137 107.52300262451172,-9.449999809265137 107.52300262451172,-9.449999809265137 C112.18199920654297,-9.449999809265137 115.95999908447266,-5.671999931335449 115.95999908447266,-1.0130000114440918z"
                          fill="rgb(170,191,207)"
                          fill-opacity="1"
                        />
                      </g>
                      <g
                        opacity="1"
                        transform="matrix(1,0,0,1,192.0399932861328,199.55299377441406)"
                      >
                        <path
                          d=" M115.95999908447266,-70.51100158691406 C115.95999908447266,-70.51100158691406 115.95999908447266,62.07400131225586 115.95999908447266,62.07400131225586 C115.95999908447266,66.73300170898438 112.18199920654297,70.51100158691406 107.52300262451172,70.51100158691406 C107.52300262451172,70.51100158691406 -107.52400207519531,70.51100158691406 -107.52400207519531,70.51100158691406 C-112.18299865722656,70.51100158691406 -115.95999908447266,66.73300170898438 -115.95999908447266,62.07400131225586 C-115.95999908447266,62.07400131225586 -115.95999908447266,-70.51100158691406 -115.95999908447266,-70.51100158691406 C-115.95999908447266,-70.51100158691406 115.95999908447266,-70.51100158691406 115.95999908447266,-70.51100158691406z"
                          fill="rgb(56,77,91)"
                          fill-opacity="1"
                        />
                      </g>
                      <g
                        opacity="1"
                        transform="matrix(1,0,0,1,271.5199890136719,119.59300231933594)"
                      >
                        <path
                          d=" M-2.8550000190734863,0 C-2.8550000190734863,-1.5770000219345093 -1.5770000219345093,-2.8550000190734863 0,-2.8550000190734863 C1.5770000219345093,-2.8550000190734863 2.8550000190734863,-1.5770000219345093 2.8550000190734863,0 C2.8550000190734863,1.5770000219345093 1.5770000219345093,2.8550000190734863 0,2.8550000190734863 C-1.5770000219345093,2.8550000190734863 -2.8550000190734863,1.5770000219345093 -2.8550000190734863,0z"
                          fill="rgb(219,28,73)"
                          fill-opacity="1"
                        />
                      </g>
                      <g
                        opacity="1"
                        transform="matrix(1,0,0,1,285.30499267578125,119.59300231933594)"
                      >
                        <path
                          d=" M-2.8550000190734863,0 C-2.8550000190734863,-1.5770000219345093 -1.5770000219345093,-2.8550000190734863 0,-2.8550000190734863 C1.5770000219345093,-2.8550000190734863 2.8550000190734863,-1.5770000219345093 2.8550000190734863,0 C2.8550000190734863,1.5770000219345093 1.5770000219345093,2.8550000190734863 0,2.8550000190734863 C-1.5770000219345093,2.8550000190734863 -2.8550000190734863,1.5770000219345093 -2.8550000190734863,0z"
                          fill="rgb(235,117,35)"
                          fill-opacity="1"
                        />
                      </g>
                      <g
                        opacity="1"
                        transform="matrix(1,0,0,1,299.0899963378906,119.59300231933594)"
                      >
                        <path
                          d=" M-2.8550000190734863,0 C-2.8550000190734863,-1.5770000219345093 -1.5770000219345093,-2.8550000190734863 0,-2.8550000190734863 C1.5770000219345093,-2.8550000190734863 2.8550000190734863,-1.5770000219345093 2.8550000190734863,0 C2.8550000190734863,1.5770000219345093 1.5770000219345093,2.8550000190734863 0,2.8550000190734863 C-1.5770000219345093,2.8550000190734863 -2.8550000190734863,1.5770000219345093 -2.8550000190734863,0z"
                          fill="rgb(255,255,255)"
                          fill-opacity="1"
                        />
                      </g>
                    </g>
                    <g
                      clip-path="url(#__lottie_element_84)"
                      style="display: none;"
                    >
                      <g
                        mask="url(#__lottie_element_130)"
                        style="display: none;"
                      >
                        <g>
                          <path />
                        </g>
                      </g>
                      <g
                        mask="url(#__lottie_element_127)"
                        style="display: none;"
                      >
                        <g>
                          <path />
                        </g>
                      </g>
                      <g
                        clip-path="url(#__lottie_element_124)"
                        style="display: none;"
                      >
                        <g>
                          <path />
                        </g>
                      </g>
                      <g
                        clip-path="url(#__lottie_element_121)"
                        style="display: none;"
                      >
                        <g>
                          <path />
                        </g>
                      </g>
                      <g
                        clip-path="url(#__lottie_element_118)"
                        style="display: none;"
                      >
                        <g>
                          <path />
                        </g>
                      </g>
                      <g
                        clip-path="url(#__lottie_element_115)"
                        style="display: none;"
                      >
                        <g>
                          <path />
                        </g>
                      </g>
                      <g
                        clip-path="url(#__lottie_element_112)"
                        style="display: none;"
                      >
                        <g>
                          <path />
                        </g>
                      </g>
                      <g
                        clip-path="url(#__lottie_element_109)"
                        style="display: none;"
                      >
                        <g>
                          <path />
                        </g>
                      </g>
                      <g
                        clip-path="url(#__lottie_element_106)"
                        style="display: none;"
                      >
                        <g>
                          <path />
                        </g>
                      </g>
                      <g
                        clip-path="url(#__lottie_element_103)"
                        style="display: none;"
                      >
                        <g>
                          <path />
                        </g>
                      </g>
                      <g
                        clip-path="url(#__lottie_element_100)"
                        style="display: none;"
                      >
                        <g>
                          <path />
                        </g>
                      </g>
                      <g
                        clip-path="url(#__lottie_element_97)"
                        style="display: none;"
                      >
                        <g>
                          <path />
                        </g>
                      </g>
                      <g
                        clip-path="url(#__lottie_element_94)"
                        style="display: none;"
                      >
                        <g>
                          <path />
                        </g>
                      </g>
                      <g
                        clip-path="url(#__lottie_element_91)"
                        style="display: none;"
                      >
                        <g>
                          <path />
                        </g>
                      </g>
                      <g
                        clip-path="url(#__lottie_element_88)"
                        style="display: none;"
                      >
                        <g>
                          <path />
                        </g>
                      </g>
                    </g>
                  </g>
                </g>
              </g>
            </svg>
          </div>
        </div>
      </div>
    </div>
  </section>
</div>
`;
